exercises/linked-list/canonical-data.json

```
@@ -0,0 +1,532 @@
+{
+  "exercise": "linked-list",
+  "comments": [
+    "In order to keep the interfae for the exercise close or equal to the     ",
+    "description.md and also satisfying the canonical-schema.json, the only   ",
+    "properties are pop, push, shift, unshift. Some tracks also implement     ",
+    "delete and count, via a Track-Inserted hints.                            ",
```

Did you mean

```suggestion
    "delete and count, via Track-Inserted hints.                              ",
```

or

```suggestion
    "delete and count, via a Track-Inserted hints.md file.                    ",
```